Sources/joyfill-components-swift/Components/ComponentViewController/APIComponents/API/JoyfillAPI.swift

```diff
@@ -230,10 +267,13 @@ func getStringValues(string: String, i: Int, j: Int, mobileView: Bool = false) {
     if componentTypeValue == "text" {
         textFieldString = string
     }
+    if componentTypeValue == "richText" {
+        jsonString = string
+    }
 }
 
 // Function to get chart and image values
-func getChartAndImageValue(valueElements: [ValueElement], j: Int) {
+func getArrayValue(valueElements: [ValueElement], j: Int) {
```

Comment:
If this is for getting the chart and image values we should name this function appropriately

Reply:
Actually I named this function as "getArrayValue" because all the components whose data is in coming in array form is fetched in this function not only chart and image.